.github/workflows/purge-old-images.yaml

```
@@ -0,0 +1,48 @@
+name: Purge old images
```

Let's give it a more explicit name/description. One suggestion below, feel free to rephrase it.

```suggestion
name: Purge test container images
```